cdk/lib/cdk-stack.ts

```diff
@@ -1,10 +1,24 @@
-import * as cdk from '@aws-cdk/core';
+import * as cdk from '@aws-cdk/core'
+import * as lambda from '@aws-cdk/aws-lambda'
+import {SqsEventSource} from '@aws-cdk/aws-lambda-event-sources'
+import * as sqs from '@aws-cdk/aws-sqs'
 
 export class CdkStack extends cdk.Stack {
   constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
-    super(scope, id, props);
-
-    // The code that defines your stack goes here
+    super(scope, id, props)
 
+    // Can add timeout, deadLetterQueue, memorySize etc. later to fine tune the created lambda
+    const queueSongLambda = new lambda.Function( this, 'QueueSongLambda', {
+      code: new lambda.AssetCode(`../queue-song-lambda/src`),
+      handler: 'Foo.foo',
```

Comment:
This'll change when I actually finish rewriting the lambda, but fine as is atm

Reply:
Yeah currently this was just to create a baseline CDK setup. None of this is set in stone even after it is deployed to AWS.